testing/correctness/tests/log_rotation/Makefile

```diff
@@ -38,7 +38,6 @@ LOG_ROTATION_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 log_rotation_tests : CUSTOM_PATH = $(SEQUENCE_WINDOW_PATH):$(VALIDATOR_PATH):$(EXTERNAL_SENDER_PATH)
 log_rotation_tets : CUSTOM_PYTHONPATH = $(SEQUENCE_WINDOW_PYTHON_PATH)
 
-build-testing-correctness-tests-log_rotation: build-testing-correctness-apps-sequence_window
```

Comment:
This whole test dir can be removed

Reply:
The `sequence_window` app is indeed already gone.  Are you referring to the `log_rotation` tests?

Reply:
Yup. The whole directory can go. Log rotation tests are part of the resilience test suite now (which is moving to the conformance tests shortly)

Reply:
Ok, removed by commit 961978b